### DISCUSSION TITLE
How to eliminate other unused classnames of tailwind from the nextjs project ?

### DISCUSSION THREAD

**Discussion by ProPhycient5** [tailwindlabs/tailwindcss#10046] [Help]
In my current process, if I want to remove unused CSS classnames, from the project. What are the steps I have to follow to do so ?

Do I need to follow the below steps as suggested in (https://tailwindcss.com/docs/optimizing-for-production)
1. run `npx tailwindcss -o build.css --minify` command
2. Add the following code in `postcss.config.js`
```
module.exports = {
  plugins: {
    tailwindcss: {},
    autoprefixer: {},
    ...(process.env.NODE_ENV === 'production' ? { cssnano: {} } : {})
  }
}
```
Will the above steps suffice to meet my requirement ?

**Comment by wongjn** [comment]
Yep! As a bonus, you could look at using [`blocklist`](https://tailwindcss.com/docs/content-configuration#discarding-classes) if needed too.

**Reply by wongjn** [reply]
> Thanks @wongjn for your response. So from the above link, I get that I have to explicitly jot down all the unused classnames within blocklist array. But don't you know think, it is time-consuming task to do so ?? I mean find the classnames which are not used and then enlist ing within `blocklist`.[^1]

It is just a "bonus" as I said so you could skip this step – you should only have a few false positives if at all.

[^1]: https://github.com/tailwindlabs/tailwindcss/discussions/10046#discussioncomment-4366228

**Comment by ProPhycient5** [comment]
Thanks @wongjn for your response. So from the above link, I get that I have to explicitly jot down all the unused classnames within blocklist array. But don't you know think, it is time-consuming task to do so ?? I mean find the classnames which are not used and then enlist ing within `blocklist`.

**Reply by wongjn** [reply]
To keep the reply chain together, I replied in the proper thread: https://github.com/tailwindlabs/tailwindcss/discussions/10046#discussioncomment-4366494
